src/reducers/index.js

```diff
@@ -15,8 +15,10 @@ import { combineReducers } from 'redux';
 
 import extensions, * as extensionSelectors from './extensions';
 import namespaces, * as namespaceSelectors from './namespaces';
+import serviceAccounts, * as serviceAccountSelectors from './serviceAccounts';
```

Comment:
move this below the secrets import, alphabetical ordering makes the imports easier to scan